src/caldavclient.cpp

```diff
@@ -36,12 +36,13 @@
 #include <QNetworkReply>
 #include <QDateTime>
 #include <QtGlobal>
+#include <QStandardPaths>
```

Comment:
Is it a leftover of something ? I don't see where it is need in the changes.

Reply:
This file was depending on an indirect include of QStandardPaths via LogMacros.h, which has now been removed.